packages/mobile/scripts/run_app.sh

```diff
@@ -63,8 +63,12 @@ sed -i.bak "s/DEV_RESTORE_NAV_STATE_ON_RELOAD=.*/DEV_RESTORE_NAV_STATE_ON_RELOAD
 # Set Firebase settings in google service config files
 ANDROID_GSERVICES_PATH="./android/app/src/debug/google-services.json"
 IOS_GSERVICES_PATH="./ios/GoogleService-Info.plist"
-sed -i.bak "s/celo-org-mobile-.*firebaseio.com/celo-org-mobile-$NETWORK.firebaseio.com/g" $ANDROID_GSERVICES_PATH
-sed -i.bak "s/celo-org-mobile-.*firebaseio.com/celo-org-mobile-$NETWORK.firebaseio.com/g" $IOS_GSERVICES_PATH
+if [ $PLATFORM = "android" ]; then
```

Comment:
I would prefer to have them both still be set in case the user is actually running both at the same time, as we sometimes do.
Can you instead check if the files exist and then run sed?

Reply:
Done :+1: